Pod/Classes/XExtensionItem.h

```diff
@@ -89,6 +126,13 @@ typedef UIImage *(^XExtensionItemThumbnailProvidingBlock)(CGSize suggestedSize,
 @property (nonatomic) XExtensionItemReferrer *referrer;
 
 /**
+ A block that will be called with the suggested size for the thumbnail image, in points, and the activity type chosen by 
+ the user. It should return an image using the appropriate scale for the screen. Images provided at the suggested size 
+ will result in the best experience.
+ */
+@property (nonatomic, copy) XExtensionItemThumbnailProvidingBlock thumbnailProvider;
```

Comment:
I’d go with `XExtensionItemThumbnailProvider`, which is more similar to Cocoa’s block type naming in general, but I don’t care too much.